inc/plugins/namespace.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Sets DID-less plugin slug as active.
+ *
+ * @package FAIR
+ */
+
+namespace FAIR\Plugins;
+
+use function FAIR\Updater\get_packages;
+
+/**
+ * Bootstrap
+ *
+ * @return void
+ */
+function bootstrap() {
+	add_filter( 'option_active_plugins', __NAMESPACE__ . '\\set_as_active' );
+	add_filter( 'wp_admin_notice_markup', __NAMESPACE__ . '\\hide_notice', 10, 3 );
+
+	// just for testing.
+	if ( ! function_exists( 'is_plugin_active' ) ) {
+		require_once ABSPATH . 'wp-admin/includes/plugin.php';
+	}
+	if ( is_plugin_active( 'git-updater/git-updater.php' ) ) {
+		wp_admin_notice( 'Git Updater is active' );
+	}
+}
+
+/**
+ * Set FAIR plugins as active using DID-less slug.
+ *
+ * @param  array $active_plugins Array of active plugins.
+ *
+ * @return array
+ */
+function set_as_active( $active_plugins ) {
+	remove_filter( 'option_active_plugins', __NAMESPACE__ . '\\set_as_active' );
+	$packages = get_packages();
+	$plugins = $packages['plugins'] ?? [];
+	foreach ( $plugins as $plugin ) {
+		if ( is_plugin_active( plugin_basename( $plugin ) ) ) {
+			$plugins[] = get_didless_slug( $plugin );
```

Comment:
Careful - `$plugin` isn't a slug in this situation, it's `trailingslashit( WP_PLUGIN_DIR ) . $file` - [Ref](https://github.com/fairpm/fair-plugin/blob/main/inc%2Fupdater%2Fnamespace.php#L44).

Reply:
Except that when I didn't have `plugin_basename()` it didn't see as active. I'll test again.

Reply:
Oh, I meant the call to `get_didless_slug( $plugin )`

Reply:
Refactored in https://github.com/fairpm/fair-plugin/pull/151/commits/5795d008f1be7f7d9a4f35a4c3fc87cce784dc0e